src/components/common/AddEditOrgPerson.vue

```diff
@@ -65,7 +65,7 @@
           >
             <!-- Person's Name -->
             <article v-if="isPerson">
-              <div class="font-weight-bold">Person's Name</div>
+              <label>Person's Name</label>
```

Comment:
Labels are better semantically than a div, and are already bolded by our app styling.

Reply:
Plus this makes this (IA) component almost the same as the Reg component.